PortableRuntimeIdentifierGraph.json not found during runtime repo build

The following error occurs when attempting to build the VMR (during the runtime repo build) with a source-built version of the SDK (stage 2 bootstrapping scenario):

```
/vmr/.dotnet/sdk/8.0.100-rc.1.23415.1/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.Sdk.FrameworkReferenceResolution.targets(90,5): error MSB4018: The "ProcessFrameworkReferences" task failed unexpectedly.
System.IO.FileNotFoundException: Could not find file '/vmr/src/runtime/artifacts/source-build/self/src/src/libraries/Microsoft.NETCore.Platforms/src/PortableRuntimeIdentifierGraph.json'.
File name: '/vmr/src/runtime/artifacts/source-build/self/src/src/libraries/Microsoft.NETCore.Platforms/src/PortableRuntimeIdentifierGraph.json'
   at Interop.ThrowExceptionForIoErrno(ErrorInfo errorInfo, String path, Boolean isDirError)
   at Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String path, OpenFlags flags, Int32 mode, Boolean failForSymlink, Boolean& wasSymlink, Func`4 createOpenException)
   at Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, UnixFileMode openPermissions, Int64& fileLength, UnixFileMode& filePermissions, Boolean failForSymlink, Boolean& wasSymlink, Func`4 createOpenException)
   at System.IO.Strategies.OSFileStreamStrategy..ctor(String path, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, Int64 preallocationSize)
   at System.IO.File.OpenRead(String path)
   at NuGet.RuntimeModel.JsonRuntimeFormat.ReadRuntimeGraph(String filePath)
   at Microsoft.NET.Build.Tasks.RuntimeGraphCache.GetRuntimeGraph(String runtimeJsonPath)
   at Microsoft.NET.Build.Tasks.ProcessFrameworkReferences.ProcessRuntimeIdentifier(String runtimeIdentifier, KnownRuntimePack selectedRuntimePack, String runtimePackVersion, List`1 additionalFrameworkReferencesForRuntimePack, HashSet`1 unrecognizedRuntimeIdentifiers, List`1 unavailableRuntimePacks, List`1 runtimePacks, List`1 packagesToDownload, String isTrimmable, Boolean addRuntimePackAndDownloadIfNecessary, Boolean wasReferencedDirectly)
   at Microsoft.NET.Build.Tasks.ProcessFrameworkReferences.AddPacksForFrameworkReferences(List`1 packagesToDownload, List`1 runtimeFrameworks, List`1 targetingPacks, List`1 runtimePacks, List`1 unavailableRuntimePacks, List`1& knownRuntimePacksForTargetFramework)
   at Microsoft.NET.Build.Tasks.ProcessFrameworkReferences.ExecuteCore()
   at Microsoft.NET.Build.Tasks.TaskBase.Execute()
   at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute()
   at Microsoft.Build.BackEnd.TaskBuilder.ExecuteInstantiatedTask(ITaskExecutionHost taskExecutionHost, TaskLoggingContext taskLoggingContext, TaskHost taskHost, ItemBucket bucket, TaskExecutionMode howToExecuteTask) [/vmr/src/runtime/artifacts/source-build/self/src/Build.proj]
```

This occurred after the merge of https://github.com/dotnet/installer/pull/17118.

Binlog: [runtime.zip](https://github.com/dotnet/source-build/files/12344149/runtime.zip)

@elinor-fung - Is this related to https://github.com/dotnet/installer/pull/17185?

--- a/eng/liveBuilds.targets
+++ b/eng/liveBuilds.targets
@@ -260,10 +260,4 @@
           DependsOnTargets="
             ResolveLibrariesRefAssembliesFromLocalBuild;
             ResolveLibrariesRuntimeFilesFromLocalBuild" />
-
-  <PropertyGroup>
-    <!-- Keep in sync with outputs defined in Microsoft.NETCore.Platforms.csproj. -->
-    <BundledRuntimeIdentifierGraphFile>$([MSBuild]::NormalizePath('$(ArtifactsBinDir)', 'Microsoft.NETCore.Platforms', 'runtime.json'))</BundledRuntimeIdentifierGraphFile>
-    <BundledRuntimeIdentifierGraphFile Condition="!Exists('$(BundledRuntimeIdentifierGraphFile)')">$([MSBuild]::NormalizePath('$(LibrariesProjectRoot)', 'Microsoft.NETCore.Platforms', 'src', 'runtime.json'))</BundledRuntimeIdentifierGraphFile>
-  </PropertyGroup>
 </Project>

